java/src/shared/model/bank/StructureBank.java

```diff
@@ -62,8 +62,10 @@ public boolean canBuildCity() {
 
     public void buildCity() {
         assert this.availableCities > 0;
+        assert this.availableSettlements < 5;
 
         availableCities--;
+        availableSettlements++;
     }
```

Comment:
I think this is code from your other PR? We'll make sure to put the other one in first.